core/controllers/acl_decorators_test.py

```diff
@@ -3662,19 +3661,15 @@ def setUp(self):
             },
             'change to state 1')
 
-        translation_suggestion = suggestion_services.query_suggestions(
-            [('author_id', self.author_id),
-             ('target_id', self.exploration_id)])[0]
-        question_suggestion = suggestion_services.query_suggestions(
-            [('author_id', self.author_id),
-             ('target_id', 'skill_123')])[0]
-        edit_state_suggestion = suggestion_services.query_suggestions(
-            [('author_id', self.author_id),
-             ('target_id', self.exploration_id)])[1]
+        translation_suggestion = suggestion_services.get_submitted_suggestions(
```

Comment:
I suggest adding assertions that there is exactly one item in the list before taking the 0th element.

Reply:
Done